doc/howto/storage_resize.md

```diff
@@ -0,0 +1,66 @@
+---
+discourse: 1333
+---
+
+# How to resize storage
+
+If you need more storage, you can increase the size of your storage pool or your storage volume.
+In some cases, it is also possible to reduce the size of a storage volume.
+
+## Grow a storage pool
+
+To increase the size of a storage pool, follow these general steps:
+
+1. Grow the size of the storage on disk.
+1. Let the file system know of the size change.
+1. Update the pool size configuration in LXD.
+
+See the specific commands for different storage drivers below.
+
+````{tabs}
+
+```{group-tab} Btrfs
+
+Enter the following commands to grow a loop-backed Btrfs pool by 5 G:
+
+    sudo truncate -s +5G <LXD_lib_dir>/disks/<pool_name>.img
+    sudo losetup -c <loop_device>
+    sudo btrfs filesystem resize max <LXD_lib_dir>/storage-pools/<pool_name>/
+    lxc storage set <pool_name> size <new_size>
+
+Replace `<LXD_lib_dir>` with `/var/snap/lxd/common/mntns/var/snap/lxd/common/lxd/` if you are using the snap, or with `/var/lib/lxd/` otherwise.
```

Comment:
Is it correct that the snap path is `.../mntns/...` here and `/var/snap/lxd/common/lxd/` for ZFS?

Reply:
I would have thought it would be `/var/snap/lxd/common/lxd/` but will defer to @stgraber on that one.

Reply:
Seems like it doesn't work with the simpler path:
```
ubuntu@vm10:~$ sudo btrfs filesystem resize max /var/snap/lxd/common/lxd/storage-pools/btrfs-pool/
ERROR: not a btrfs filesystem: /var/snap/lxd/common/lxd/storage-pools/btrfs-pool/
```

Reply:
When interacting with files in the `disks` directory, we should say to use `/var/snap/lxd/common/lxd/disks/`.
When dealing with the mounted pool, we need to use `/var/snap/lxd/common/mntns/var/snap/lxd/common/lxd/storage-pools/NAME`.

Sounds like the documentation is currently correct.